### DISCUSSION TITLE
Split Managed Metadata Values

### DISCUSSION THREAD

**Discussion by m3d0c** [microsoft-search/pnp-modern-search#3332] [General]
Hi I am trying to figure out how split and render Managed metadata. I have created associated a refinable string to my managed metadata column and I can render the value 

![image](https://github.com/microsoft-search/pnp-modern-search/assets/34741272/be38cea5-4765-432e-b258-a888fe5a1299)

But from the previous discussion around split, this does not render anything

`split slot item @root.slots.FunctionUnit ";"` Where FunctionUnit is mapped to the refinablestring value.

I have also tried 

{{#each (split slot item @root.slots.FunctionUnit ";") as |value|}}
            <p>{{value}}</p>
{{/each}}

But nothing renders.

Any help would be appreciated.

**Comment by kasperbolarsen** [comment]
And root.slots.FunctionUnit points to the Refinablestring you mentioned?

**Reply by m3d0c** [reply]
Yes as you can see in the screenshot, the value that is shown under the “Test” word is the FunctionUnit value.

**Reply by kasperbolarsen** [reply]
I am spliting a multivalue tax property here

 <td>{{#if (slot item @root.slots.TaxMulti)}}
              {{#each (split (slot item @root.slots.TaxMulti) ";") as |tag| }}
                  {{last (split tag "|")}}
                  <br>
              {{/each}}    
      {{/if}}

And it is displayed as this. Hopes it is something this you are asking about.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/e4047766-4458-404c-9fcc-1dfed742ad2e)

**Reply by m3d0c** [reply]
Thank you @kasperbolarsen. That worked great. Is there a reference guide on how on handlebars and how can you add logic to the code?

**Reply by kasperbolarsen** [reply]
As far as I know, a PnP Modern Search specific handlebars samples library does not exist.
I am not sure that handlebar templates will get much TLC as it sure looks like Adaptive Cards are the future.

**Reply by m3d0c** [reply]
Is there a guide on how to use Adaptive cards with similar logic?
